tests/unit/test_util.py

```diff
@@ -0,0 +1,23 @@
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from cylc.flow.util import deserialise
+
+
+def test_deserialise():
+    actual = deserialise('["2", "3"]')
+    expected = {'2', '3'}
+    assert actual == expected
```

Comment:
The doctest is already running this example?

Reply:
Woops that is a mistake, thanks, I'll remove the doctest. Can't test set comparison in doctest as it does a literal check.

Reply:
Aah.